snuba/datasets/dataset.py

```diff
@@ -1,13 +1,14 @@
 from typing import Optional, Mapping, NamedTuple, Sequence, Tuple, Union
 
+from snuba.clickhouse.escaping import escape_identifier
```

Comment:
I don't know if this is a good thing to be excited about, but I do like how this makes the coupling here more explicit.

Reply:
I wanted to just move those method into the formatter, but then I would have util depending on the formatter, which would likely introduce a circular dependency as soon as the formatter needed something from util (which would be expected).

Reply:
Let's not forge that the only dependency is in column_expr, thus it is going to disappear as soon the new query processing takes over.

Reply:
Good point. I did mean this more generically, and as a positive, not as a criticism: I think that moving this (and other ClickHouse-specific functionality) to a module that literally says "`clickhouse`" in the module path makes it more clear where this coupling is so we can remove it in the future.